### PR TITLE
Documentation for project import via setup.json #425

### LINKED ISSUE #425: Adapt documentation for setup.json for project import
In ces-setup there is now an option within the unattended setup to import projects. This option should be considered in the documentation for setup.json.

### PR DESCRIPTION
**Pull request by CloudFlo2312**
Resolves #425.
Contains only changes to the documentation.

### CODE REVIEW

File: docs/docs/user-guide/setup-json_de.md [hunk 1]
```diff
@@ -295,6 +295,75 @@ Eigenschaften:
...
+      "password":"<password>",
+      "type":"Git",
+      "location":"https://github.com/cloudogu/demo-content-petclinic",
+      "reference":{
+        "name":"0.0.1",
+        "type":"Tag"
+      }
+    }
+  ]
+  "completed":true
```
Comment (sklein94) [inline docs/docs/user-guide/setup-json_de.md:359]:
```suggestion
##### Example:
````
"projects":{
  "import":[
    {
      "username":"<username>",
      "password":"<password>",
      "type":"Git",
      "location":"https://github.com/cloudogu/demo-content-petclinic",
      "reference":{
        "name":"0.0.1",
        "type":"Tag"
      }
    }
  ],
  "completed":true
}
```

File: docs/docs/user-guide/setup-json_de.md [hunk 2]
```diff
@@ -295,6 +295,75 @@ Eigenschaften:
...
+````
```
Comment (sklein94) [inline docs/docs/user-guide/setup-json_de.md:360]:
```suggestion
````
"projects":{
  "import":[
    {
      "username":"<username>",
      "password":"<password>",
      "type":"Git",
      "location":"https://github.com/cloudogu/demo-content-petclinic",
      "reference":{
        "name":"0.0.1",
        "type":"Tag"
      }
    }
  ],
  "completed":true
}
````
```

File: docs/docs/user-guide/setup-json_de.md [hunk 3]
```diff
@@ -295,6 +295,75 @@ Eigenschaften:
...
+      "username":"<username>",
+      "password":"<password>",
+      "type":"Git",
+      "location":"https://github.com/cloudogu/demo-content-petclinic",
+      "reference":{
+        "name":"0.0.1",
+        "type":"Tag"
+      }
+    }
+  ]
```
Comment (sklein94) [inline docs/docs/user-guide/setup-json_de.md:358]:
```suggestion
  ],
```

File: docs/docs/user-guide/setup-json_de.md (hunk 2 above)
Comment (sklein94) [inline docs/docs/user-guide/setup-json_de.md:360]:
```suggestion
  "completed":true
}
````
```

File: docs/docs/user-guide/setup-json_de.md (hunk 1 above)
Comment (sklein94) [inline docs/docs/user-guide/setup-json_de.md:359]:
```suggestion
  "completed":true
 }
```

File: docs/docs/user-guide/setup-json.md [hunk 4]
```diff
@@ -291,6 +291,75 @@ Properties:
...
+      "username":"<username>",
+      "password":"<password>",
+      "type":"Git",
+      "location":"https://github.com/cloudogu/demo-content-petclinic",
+      "reference":{
+        "name":"0.0.1",
+        "type":"Tag"
+      }
+    }
+  ]
```
Comment (sklein94) [inline docs/docs/user-guide/setup-json.md:355]:
```suggestion
  ],
```

File: docs/docs/user-guide/setup-json.md [hunk 5]
```diff
@@ -291,6 +291,75 @@ Properties:
...
+  "completed":true
```
Comment (sklein94) [inline docs/docs/user-guide/setup-json.md:356]:
```suggestion
  "completed":true
},
```

**Review by Subaro — APPROVED** [review]

**Review by Subaro — APPROVED** [review]

**Pull request merged**